Taskfile.yml

```diff
@@ -21,7 +22,7 @@ vars:
   TIFF_ARCHIVE: libtiff.a
   TIFF_CONFIG: "{{.TIFF_SRC}}/configure"
 
-  TESSERACT_VERSION: 4.1.3
+  TESSERACT_VERSION: 5.0.1
```

Comment:
Not really comfortable merging this until I can check if there are any API breakages for the use in SwiftyTesseract